modules/sql-engine/src/main/java/org/apache/ignite/internal/sql/engine/externalize/RelInputEx.java

```diff
@@ -38,8 +40,17 @@
     /**
      * Returns table by its id.
      *
-     * @param tag Tag under which id is serialised.
+     * @param tag Tag under which id is serialized.
      * @return A table with given id.
      */
     RelOptTable getTableById(String tag);
+
+    /**
+     * Check that appropriate index is available.
+     *
+     * @param tag Tag represents id serialization.
+     * @param idxName Index name.
```

Comment:
I believe the index name doesn't matter here

Reply:
What kind of information will obtain user in case of error in such a case ? I think that UUID is also not for user eyes but it`s discussable.

Reply:
Exception thrown by this method is an internal one. Both the name and the Id is not for the user. I believe, the exception should be converted to some general "Unexpected error, please retry the query" exception

Reply:
fixed